subsys/infinispan/src/main/java/org/commonjava/indy/subsys/infinispan/CacheProducer.java

```diff
@@ -179,17 +180,13 @@ private void startCacheManager( String configFile, Boolean isCluster )
                 try
                 {
                     logger.info( "Using CLASSPATH resource Infinispan configuration:\n\n{}\n\n", resourceStr );
+                    mgr = new DefaultCacheManager(
+                            new ByteArrayInputStream( resourceStr.getBytes( StandardCharsets.UTF_8 ) ) );
                     if ( isCluster )
                     {
-                        clusterCacheManager = new DefaultCacheManager(
-                                        new ByteArrayInputStream( resourceStr.getBytes( StandardCharsets.UTF_8 ) ) );
-                        clusterCacheManager.startCaches( String.join( ",",  clusterCacheManager.getCacheNames()) );
-                    }
-                    else
-                    {
-                        cacheManager = new DefaultCacheManager(
-                                        new ByteArrayInputStream( resourceStr.getBytes( StandardCharsets.UTF_8 ) ) );
+                        mgr.startCaches( String.join( ",",  clusterCacheManager.getCacheNames()) );
```

Comment:
Does this only run when a ispnConf file is not supplied?

Reply:
From code here it seems like this. I'm not quite sure why it acts like this as I did not change the legacy code logic here. 
@sswguo  do you know why this perform like this?

Reply:
Not sure, just added the cluster logic in the else section. Im curious why we checks the classpath one and merge it but not just put all the things in one place like in /etc/... instead. :)